components/list/list-item-placement-marker.js

```diff
@@ -0,0 +1,29 @@
+import { html, LitElement } from 'lit-element/lit-element.js';
+
+class ListItemPlacementMarker extends LitElement {
+	static get properties() {
+		return {
+			color: { type: String },
+			displayed: {type: Boolean}
+		};
+	}
+
+	constructor() {
+		super();
+		this.color = "blue";
+		this.strokeWidth = 4;
+		this.radius = this.strokeWidth * 2;
+		this.height = this.radius * 2 + this.strokeWidth * 2;
+	}
+
+	render() {
+	  return this.displayed ? html`
+		<svg height="${this.height}"  width="100%">
+		  	<circle cx="${this.strokeWidth + this.radius}" cy="${this.strokeWidth + this.radius}" r="${this.radius}" stroke="${this.color}" stroke-width="${this.strokeWidth}" fill="white" />
+	  		<line x1="${this.radius*2 + this.strokeWidth}" y1="${this.height/2}" x2="98%" y2="${this.height/2}" stroke="${this.color}" stroke-width="${this.strokeWidth}" stroke-linecap="round"/>
```

Comment:
x2="98%" - not sure how to improve this, If I  set x2 = 100%, the linecap is cut off"

Reply:
For this, I would recommend using a third shape as in the Prototype [here](https://github.com/BrightspaceHypermediaComponents/activities/blob/majones/PROTOTYPE/dragList/components/d2l-activity-collection-editor/d2l-activity-collection-list-item.js#L363)
Just place the line-cap piece overlapping the existing line so we get that nice rounded edge

Reply:
Yes, what zina said about the third shape is the only way to do it.
```
<div class="${htmlClass}">
<svg viewBox="0 0 12 12" class="d2l-list-drag-indicator-circle">
	<circle cx="6" cy="6" r="4" fill="none"/>
</svg>
<svg class="d2l-list-drag-indicator">
	<line x1="0" y1="50%" x2="100%" y2="50%" />
</svg>
<svg viewBox="0 0 5 12" class="d2l-list-drag-indicator-linecap">
          <line x1="-5" y1="50%" x2="0" y2="50%" />
</svg>
```
I also found it helpful to be more dynamic to split them into separate svgs. This allows the user of this component to define the width without any stretching of the circle or the round end cap.

Reply:
I notice there are some off effects with the width - specifically the svg is overlapping the border in my mock-up, but without the border, it is overflowing. (zoomed screenshot)

![image](https://user-images.githubusercontent.com/9042472/83660920-7599a300-a593-11ea-936e-0c1bb047c189.png)

Without the border, the edge of the circle appears to get cut off. (zoomed screenshot)

![image](https://user-images.githubusercontent.com/9042472/83661092-aaa5f580-a593-11ea-9822-92b3a31cfc6e.png)

I also notice some odd things with the width of the line - like it doesn't always go 100% of it's container the way I expect it to.  (not zoomed screenshot, svg on top, div on bottom)

![image](https://user-images.githubusercontent.com/9042472/83661411-1a1be500-a594-11ea-9145-f61bffa4ea9f.png)


Was a `div` sol'n considered?  It seems simpler to me.

```
import { css, html, LitElement } from 'lit-element/lit-element.js';
import { RtlMixin } from '../../mixins/rtl-mixin.js';

class ListItemPlacementMarker extends RtlMixin(LitElement) {

	static get styles() {
		return css`
			:host {
				display: block;
				position: relative;
				line-height: 0;
			}

			:host([hidden]) {
				display: none;
			}

			.d2l-list-drop-marker-circle {
				border: 3px solid var(--d2l-color-celestine);
				border-radius: 6px;
				box-sizing: border-box;
				display: inline-block;
				height: 11px;
				width: 11px;
			}

			.d2l-list-drop-marker-line {
				border-bottom: 2px solid var(--d2l-color-celestine);
				border-top: 1px solid var(--d2l-color-celestine);
				border-radius: 1px;
				position: absolute;
				top: 4px;
				left: 10px;
				width: calc(100% - 10px);
			}
		`;
	}

	render() {
		return html`
			<div class="d2l-list-drop-marker-circle"></div>
			<div class="d2l-list-drop-marker-line"></div>
		`;
	}

}

customElements.define('d2l-list-item-placement-marker-2', ListItemPlacementMarker);

```

Reply:
@dbatiste thank you for catching the cut-off issue. I've tried to implement your exampla and found another problem - line some times is split when zooming page with ctrl+/-
I've adjusted margins in "my approach" and it looks better.
![image](https://user-images.githubusercontent.com/14826663/83898731-4d41ae00-a725-11ea-9ca6-a2eff4419950.png)